pandas/tests/indexing/multiindex/test_loc.py

```diff
@@ -695,3 +695,17 @@ def test_loc_getitem_index_differently_ordered_slice_none():
         columns=["a", "b"],
     )
     tm.assert_frame_equal(result, expected)
+
+
+def test_loc_getitem_drops_levels_for_one_row_dataframe():
+    # GH#10521
+    mi = MultiIndex.from_arrays([["x"], ["y"], ["z"]], names=["a", "b", "c"])
+    df = DataFrame({"d": [0]}, index=mi)
+    expected = df.copy()
+    result = df.loc["x", :, "z"]
+    tm.assert_frame_equal(result, expected)
+
+    ser = Series([0], index=mi)
+    result = ser.loc["x", :, "z"]
+    expected = Series([0], index=Index(["y"], name="b"))
+    tm.assert_series_equal(result, expected)
```

Comment:
big picture, why isnt `ser_expected = frame_expected["d"]`?

Reply:
Series drops levels of MultiIndex (for example #6022), while DataFrame should keep them. The current behavior is, that DataFrame keeps them all the times except when it has one row.

Reply:
is https://github.com/pandas-dev/pandas/pull/38150#issuecomment-752233136 saying that you plant to change the Series behavior to match DataFrame?

Reply:
No, the comment was meant to clarify why we have to check for ndim==1